### ISSUE TITLE
Components added in layout via shortcodes don’t get mounted

### ISSUE THREAD

**Issue by NK-WebDev** [Elderjs/elderjs#78]
Hi, I'm using the elderjs/image plugin and I can't figure out how to render the picture element in my svelte component

this is my component
```svelte
<script lang="ts">
  export let helpers;
</script>

<style lang="scss">
  img {
    margin: 0.2rem;
    max-height: 4.5rem;
    padding: var(--spacing-0);
    @media (max-width: 40em) {
      max-height: 3rem;
      padding: 0;
    }
  }
</style>

<a href={helpers.permalinks.home({slug: '/'})}>
  {
    @html helpers.shortcode({name: 'picture',props:{src:'/img/logo.png'}})
  }

  <!-- {@html helpers.picture('/img/logo.png', {maxWidth: 2000, class:'', alt: ''})} -->
</a>

```

and this is being rendered instead {{picture src='/img/logo.png'/}}

  

I also tried the helpers.picture way but I'm getting helpers.picture is not a function error message If I do that.

**Comment by nickreese** [comment]
@NK-WebDev this was fixed in v1.1.6 that was just published.

**Comment by NK-WebDev** [comment]
@nickreese Thanks for fixing it so quickly  :clap: ,I updated elder.js and tried the image plugin shortcode and it's working now.
